pkg/syncer/k8scloudoperator/placement.go

```diff
@@ -57,8 +57,11 @@ const (
 	//resource name to get storage class
 	resourceName = "storagepools"
 	// storagePool type name for vsan-direct
-	vsanDirect = "vsanD"
-	vsanSna    = "vsan-sna"
+	vsanDirect       = "vsanD"
```

Comment:
Not sure how this is used. If this is user-visible can we expand `vsanD` to `vsanDirect`?

Reply:
Its not user visible and the placement engine is the "user" of this label from the storage pool - we can't change it to anything other than what's on the storage pool.